src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -241,7 +241,7 @@ define(function (require, exports, module) {
      * @return {string}
      */
     function _resolveUrl(path) {
-        return _server && _server.pathToUrl(path);
+        return _server && path;
```

Comment:
What does this change mean for other places where `_resolveUrl` is called?  I'm especially wondering about how the `LiveDoc` ctor uses it?  If you do make this change, it should just be `return path;` without the `_server` check.